SPECS/postgresql/postgresql.spec

```diff
@@ -64,10 +70,11 @@ The postgresql-devel package contains libraries and header files for
 developing applications that use postgresql.
 
 %prep
-%setup -q
+%autosetup -p1 -n %{name}-%{version}
```

Comment:
nit: is the `-n %{name}-%{version}` required here? I think that's just the default.

Reply:
you're right. I removed it